.github/codeql/custom-queries/dos/UnboundedArrayInRoute.ql

```diff
@@ -0,0 +1,78 @@
+/**
+ * @name Unbounded array in schema validation
+ * @description Detects schema.arrayOf() calls without maxSize constraint,
+ *              which could lead to Denial of Service (DoS) vulnerabilities
+ *              when used to validate untrusted input.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 6.5
```

Comment:
@elastic/kibana-security discussion topic.

A severity of `6.5` places this in the `medium` bucket (source: https://github.blog/changelog/2021-07-19-codeql-code-scanning-new-severity-levels-for-security-alerts/#about-security-severity-levels). Do we feel this is reasonable, or should we look to adjust?